src/main/java/org/testng/TestNG.java

```diff
@@ -821,7 +822,9 @@ public void addInvokedMethodListener(IInvokedMethodListener listener) {
   }
 
   public Set<IReporter> getReporters() {
-    return m_reporters;
+    //This will now cause a different behavior for consumers of this method because unlike before they are no longer
```

Comment:
If you would want, I can add this at the comments also, but I thought that this comment would add value in the code, because the moment someone experiences a different behaviour (as documented by the comment), they would be looking at the code and then read this comment. If anyone would want to tie this change back to a commit, then a `git blame` would also do the trick. But I will add this to the comment message as well.